Grupo_3_TP_Integrador/src/dominio/Cuenta.java

```diff
@@ -0,0 +1,97 @@
+package dominio;
+
+import java.math.BigDecimal;
+import java.sql.Date;
+import java.time.LocalDate;
+
+public class Cuenta {
+	private int id;
+    private Cliente cliente;
+    private TipoCuenta tipoCuenta;
+    private Date fechaCreacion;
+    private Long numeroCuenta;
+    private String cbu;
+    private BigDecimal saldo;
+    private boolean activa;
+	    
+    public Cuenta() {
+        this.fechaCreacion = Date.valueOf(LocalDate.now());
+        this.saldo = new BigDecimal("10000.00");
+        this.activa = true;
+    }
+    
+    public Cuenta(int id, Cliente cliente, TipoCuenta tipoCuenta, Long numeroCuenta, String cbu) {
```

Comment:
noté que el constructor no incluye el atributo saldo. 
quizás es a propósito pero x las dudas dejo comentario.

Reply:
Misma observación: no incluye saldo ni fecha, y en la BD son not null